liip/imagine-bundle/1.8/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Liip\\ImagineBundle\\LiipImagineBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%/"
+    },
+    "copy-from-package": {
+    	"Resources/config/routing.yaml": "%ETC_DIR%/routing/imagine.yaml"
```

Comment:
Hm, so changing this (per your suggestion) doesn't actually cause this recipe to work, though. Any idea why the recipe doesn't apply the configuration file or the routing file, using either method?